packages/mantine/src/components/table/TableHeader.tsx

```diff
@@ -45,7 +45,7 @@ export const TableHeader: FunctionComponent<TableHeaderProps> = ({
             className={classes.root}
             {...others}
         >
-            {multiRowSelectionEnabled && selectedRows.length > 0 ? (
+            {multiRowSelectionEnabled && !disableRowSelection && selectedRows.length > 0 ? (
```

Comment:
I removed the button because if it is clicked on, it modifies and clears the selected rows. In this case, should I simply remove the clear icon and prevent the onClick method?

Reply:
It should be in this state and no action should be available.

![image](https://github.com/coveo/plasma/assets/966550/a8e990eb-01f8-499b-be77-07e42d86ce7c)

Reply:
That makes sense, I'll add the necessary changes